tests/test_modeling_common.py

```diff
@@ -2521,15 +2521,17 @@ def test_disk_offload(self):
             base_output = model(**inputs_dict_class)
 
             model_size = compute_module_sizes(model)[""]
-            max_size = int(self.model_split_percents[1] * model_size)
             with tempfile.TemporaryDirectory() as tmp_dir:
                 model.cpu().save_pretrained(tmp_dir)
 
-                max_memory = {0: max_size, "cpu": max_size}
                 with self.assertRaises(ValueError):
+                    max_size = int(self.model_split_percents[0] * model_size)
```

Comment:
we need `0.5` to get the desired error being raised

Reply:
This is 0.6 now ;-p